### PR TITLE
[ECUX-314] Add UD icons, setup tests

### PR DESCRIPTION
**Pull request by dgrishajev**
This PR adds all common icons used on Unstoppable Domains website, as well as their stories

New icons are presented in the updated Storybook: https://unstoppabledomains.github.io/UI-Kit/?path=/story/components-icons--crypto-icons

Also added testing infra and readme badges.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unstoppabledomains/ui-kit",
-  "version": "0.2.3",
+  "version": "0.2.4-alpha.0",
```
Comment (iurevych) [inline package.json:3]: Should we simply name it 0.2.4?

File: src/icons/crypto/Blocks.tsx [hunk 2]
```diff
@@ -0,0 +1,1832 @@
+import type {SvgIconProps} from '@mui/material';
+import {SvgIcon} from '@mui/material';
+import React from 'react';
+
+const Blocks = React.forwardRef<
+  SVGSVGElement,
+  {
+    iconRef?: React.ForwardedRef<SVGSVGElement>;
+  } & SvgIconProps
```
Comment (iurevych) [inline src/icons/crypto/Blocks.tsx:9]: This is a huge icon, can we ask the designer to optimize it for 32x32 px size?

**Review by iurevych — APPROVED** [review]
Looking good!

Have a look at the paths conflict for certain icons:
Icons - General Icons ⋅ Storybook 2022-09-28 22-23-09

Also I'd make icon previews more usable, it is currently hard to see how to import a certain icon:
Icons - General Icons ⋅ Storybook 2022-09-28 22-28-01

File: package.json (hunk 1 above)
Comment (dgrishajev) [reply package.json:3]: Yep, it'll become 0.2.4 once these changes are stabilized and ready to be merged. `-alpha.*` is used for incrementing test releases indicating that they're not stable yet. Also, it allows us to publish a version to npm and use the new version in the [website's PR](https://github.com/unstoppabledomains/unstoppable-domains-website/pull/5840) without breaking the checks.

File: src/icons/crypto/Blocks.tsx (hunk 2 above)
Comment (dgrishajev) [reply src/icons/crypto/Blocks.tsx:9]: Alex created the 36x36 icon for me and I added it to the `crypto/Blocks` path as `Blocks36x36` name and to the storybook as well

Comment (dgrishajev) [reply src/icons/crypto/Blocks.tsx:9]: Also increased the icon containers width, set icon names font size to 14px, and added a tooltip to each icon with both import pathes (default and named) with the Copy buttons. I wasn't sure how to add a font to Storybook and haven't had a chance to figure it out yet.

Comment (dgrishajev) [reply src/icons/crypto/Blocks.tsx:9]: https://unstoppabledomains.github.io/UI-Kit/?path=/story/components-icons--crypto-icons

Comment (dgrishajev) [reply src/icons/crypto/Blocks.tsx:9]: I'd postpone improving font families for later if it's not critical

**Pull request merged**